src/app/frontend/common/components/uploadfile/component.ts

```diff
@@ -43,7 +42,10 @@ export class UploadFileComponent {
         content,
       } as KdFile);
     };
-
-    reader.readAsText(file);
+    if (file instanceof ArrayBuffer) {
+      // throw an error, 'cause you can't handle this
```

Comment:
Can you explain this?

Reply:
https://stackoverflow.com/a/53292493

Reply:
Yeah, but anything more? Why did you leave a comment instead of doing something inside `if`?

Reply:
i was searching how you are showing errors in the app sorry i forgot to mention WIP tag

Reply:
Throw an error with some message instead of keeping comment.

Reply:
Ok sorry for the mistake, i will update the code in sometime

Reply:
I have reverted the code and displayed mat dialog in case of error! for now @floreks